### DISCUSSION TITLE
PvP Feedback

### DISCUSSION THREAD

**Discussion by ghost** [FakeFishGames/Barotrauma#14287] [Unstable feedback]
This is from experience after hosting around an hour and a half of PvP games.
# King of the Hull
- It didn't work.

# Deathmatch
- Players have no reason to pick any class besides security or captain. Security gets a baton and a knife, and the one lucky captain player per team gets a free revolver.
- NPCs on the station were unpleasant, unarmed players could flee an attacker and use them as a meatshield. For certain AI, they would shoot back at the attacker.
- Too many weapon crates with too little equipment. Each crate felt pretty worthless and it was a slog sifting through a huge number of them.

# Sub V Sub
- There's no real balancing if teams want to pick different subs, IE, a barsuk vs a typhon 2.
- Meds are very scarce: a team of 3 on a herja barely had enough meds to go around.

**Comment by ubertpendragon** [comment]
# King of the Hull

* I don't see it working if regular subs are allowed. Feels like the mod is designed around respawn shuttles.

# Sub V Sub

* There's a scramble for the medical and weapons cabinets and then you have to somehow distribute meds and weapons. **Cut down on prep time by giving everyone a "bare minimum" loadout** (1x Revolver 6x bullets 3x bandages 2x morphine 1x saline)

* Empty hardpoints on some subs lead to silly situations. Typically one sub has a no gun on the bottom, and gets destroyed by the sub that has a gun on the top below it. _Honestly the game mode is better with custom subs specifically made for pvp_

* Different suit colors for different teams would be nice, not sure how to implement that though

* Option to respawn as monsters. It's fun, trust me.

**Comment by Regalis11** [comment]
This is somewhat outdated now, closing in favor of the newer discussion: https://github.com/FakeFishGames/Barotrauma/discussions/14495
